service/src/main/resources/application.yml

```diff
@@ -18,7 +18,7 @@
 # Server properties
 ###############################################################################
 server:
-  port: 8080
+  port: 8081
```

Comment:
why?

Reply:
sory, this shouldn't have been in the commit